salt/beacons/btmp.py

```diff
@@ -26,7 +26,7 @@
 
 __virtualname__ = 'btmp'
 BTMP = '/var/log/btmp'
-FMT = 'hi32s4s32s256shhiii4i20x'
+FMT = b'hi32s4s32s256shhiii4i20x'
```

Comment:
Your edits in this file look good, but I think I found a couple bugs that pre-dated your changes (and thus I cannot add a comment down there since they don't involve your diff). See [this line](https://github.com/saltstack/salt/pull/45238/files#diff-6923ee45f755c24a8dbd1ab23dd569efR195), and the following block:

```python
if isinstance(event[field], bytes):
    event[field] = event[field].decode()
    event[field] = event[field].strip('b\x00')
else:
    event[field] = event[field].strip('\x00')
```

Firstly, `'b\x00'` looks to be a typo, introduced in 6fe02e3c6c07fd1db9f5fa432c82c444734b4b34. I would re-write this block as:

```
if isinstance(event[field], bytes):
    event[field] = salt.utils.stringutils.to_unicode(event[field])
event[field] = event[field].strip('\x00')
```

The act of decoding will give you a unicode string, not bytes and there should thus be no `b` prefix. Therefore, we only should need the one `strip` statement.